src/Version.hs

```diff
@@ -0,0 +1,14 @@
+-- Hoff -- A gatekeeper for your commits
+-- Copyright 2022 Channable
+-- Copyright 2016 Ruud van Asseldonk
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- A copy of the License has been included in the root of the repository.
+module Version (version) where
```

Comment:
We can avoid this module, and thus remove one of the places we would now have to keep in-sync, by using the [`version :: Version` export from the cabal-generated `Paths_hoff` module](https://cabal.readthedocs.io/en/3.4/cabal-package.html#accessing-the-package-version)

Reply:
Note, from a different section of the cabal users guide:

> If you decide to import the `Paths_pkgname` module then it must be listed in the [other-modules](https://cabal.readthedocs.io/en/3.4/cabal-package.html#pkg-field-other-modules) field just like any other module in your package and on [autogen-modules](https://cabal.readthedocs.io/en/3.4/cabal-package.html#pkg-field-autogen-modules) as the file is autogenerated.

Reply:
Don't forget to update `package/check-version.sh`

Reply:
Mmm... interesting... :thinking:   Lemme try to do that.

Reply:
It worked.  Great suggestion!  :smile: